<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Hebrew</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Hebrew</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos21">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos31">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos41">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos12">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos22">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos32">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos42">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos43">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos33">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos23">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos44">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos34">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos24">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pos14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
